src/main/java/org/fogbowcloud/app/api/http/services/JobService.java

```diff
@@ -57,6 +57,10 @@ public String addJob(String jdfFilePath, User owner) throws CompilerException, I
         return this.iguassuFacade.addJob(jdfFilePath, owner);
     }
 
+    public void updateJob(JDFJob job){
```

Comment:
do we need really need both a service (it seems it is required, ok), a facade and a controller? Again, no need to refactor it now but, at least, drop a comment to do it latter

Reply:
we decided to keep it as it is today